File: 151002/Haiduk/RV/RV/Lab1/Models/News.cs
namespace Lab1.Models
{
    public class News : BaseModel
    {
        public long CreatorId { get; set; }

        public Creator Creator { get; set; }

        public string Title { get; set; }

        public string Content { get; set; }

        public DateTime Created { get; set; }

        public DateTime Modified { get; set; }
    }
}

File: 1510031/Shamenov/project/lab1/src/main/java/by/bsuir/dc/lab1/entities/News.java
package by.bsuir.dc.lab1.entities;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.math.BigInteger;
import java.sql.Date;

@Setter
@Getter
@NoArgsConstructor
@AllArgsConstructor
public class News {

    private BigInteger id;

    private BigInteger editorId;

    private String title;

    private String content;

    private Date created;

    private Date modified;
}
